Source/Core/Core/AchievementManager.cpp

```diff
@@ -73,9 +79,19 @@ void AchievementManager::Init()
     m_queue.Reset("AchievementManagerQueue", [](const std::function<void()>& func) { func(); });
     m_image_queue.Reset("AchievementManagerImageQueue",
                         [](const std::function<void()>& func) { func(); });
+
+#ifdef RC_CLIENT_SUPPORTS_RAINTEGRATION
+    // attempt to load the integration DLL from the directory containing the main client
+    // executable in x64 build, will look for RA_Integration-x64.dll, then RA_Integration.dll. in
+    // non-x64 build, will only look for RA_Integration.dll
+    std::wstring exe_path = UTF8ToWString(File::GetExePath());
+    rc_client_begin_load_raintegration(m_client, exe_path.c_str(), reinterpret_cast<HWND>(hwnd),
+                                       "MyClient", "1.0", LoadIntegrationCallback, NULL);
```

Comment:
Any particular reason why this is using "MyClient" rather than a more descriptive name like "Dolphin"?